src/Data/mockData.js

```diff
@@ -0,0 +1,66 @@
+export const posts = [
+  {
+    userID: 2,
+    userName: 'Betsy',
+    title: 'Too many carrots!',
+    description: '9 carrots, all are good condition.',
+    location: 'Denver, CO',
+    distance: '~3.4 miles',
+    image: 'https://images.unsplash.com/photo-1522184216316-3c25379f9760?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80'
+  },
+  {
+    userID: 1,
+    userName: 'Jimmathy',
+    title: 'Too many pototoes!',
+    description: '33 potatoes, most are good condition.',
+    location: 'Longmont, CO',
+    distance: '~1.2 miles',
+    image: 'https://images.unsplash.com/photo-1573196444577-af471298e034?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=750&q=80'
+  },
+  {
+    userID: 3,
+    userName: 'Jetski',
+    title: 'Too much basil!',
+    description: '2 lb of basil, all in good condition.',
+    location: 'Broomfield, CO',
+    distance: '~5.1 miles',
+    image: 'https://images.unsplash.com/photo-1500420254515-0faefa2dac99?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=334&q=80'
+  },
+  {
+    userID: 2,
+    userName: 'Betsy',
+    title: 'Too many onions!',
+    description: '4 onions, all are good condition.',
+    location: 'Denver, CO',
+    distance: '~3.4 miles',
+    image: 'https://images.unsplash.com/photo-1467019972079-a273e1bc9173?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80'
+  }
+]
+
+export const user = {
+  name: 'Duckie',
+  id: 2,
+  location: 'Boulder, CO',
+  bio: 'Just a small-town girl, livin in a lonely wooorld.',
+  numberOfShares: 5,
+  posts: [
+    {
+      userID: 2,
+      userName: 'Duckie',
+      title: 'Too many onions!',
+      description: '4 onions, all are good condition.',
+      location: 'Denver, CO',
+      distance: '~3.4 miles',
+      image: 'https://images.unsplash.com/photo-1467019972079-a273e1bc9173?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80'
+    },
+    {
+      userID: 2,
+      userName: 'Duckie',
+      title: 'Too many carrots!',
+      description: '9 carrots, all are good condition.',
+      location: 'Denver, CO',
+      distance: '~3.4 miles',
+      image: 'https://images.unsplash.com/photo-1522184216316-3c25379f9760?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80'
+    }
+  ]
+}
```

Comment:
Thanks for adding some mock data Cole! This will be good to have so we can display the feed and profile info.

Reply:
LOVE the mock data you put together here!
I'm wondering if the location value may be better represented as an **object**, based on data I've seen from previous use of a geolocating API, but we can confirm that data type as we approach functionality to access that data